### DISCUSSION TITLE
styling issue

### DISCUSSION THREAD

**Discussion by TahaShek** [mdbootstrap/TW-Elements#1847] [Chat about dev stuff]
how to implement 2 css styling in the select   <select data-te-select-init
  data-te-class-select-option-text="text-[12px]"data-te-class-form-outline="relative bg-red-600"

  [(ngModel)]="selectedColumn">
  <option   value="">All Columns</option>
  <option

  *ngFor="let header of headers" [value]="header.fieldName">{{ header.heading }}</option>
 </select>

**Comment by juujisai** [comment]
When changing classes with `data-te-class` it replaces all of the classes that were added with the specific attribute / key. For example

- `data-te-class-select-option-text="group text-[12px]"`
- `data-te-class-form-outline="relative  bg-red-600"`
